src/acceptance-test/java/org/zalando/nakadi/webservice/hila/HilaAT.java

```diff
@@ -280,4 +285,43 @@ public void testGetSubscriptionStat() throws Exception {
                 .content(new StringContains(JSON_TEST_HELPER.asJsonString(new ItemsWrapper<>(subscriptionStats))));
     }
 
+    @Test(timeout = 10000)
+    public void whenConsumerIsBlocked429() throws Exception {
+        final FloodService.Flooder flooder =
+                new FloodService.Flooder(eventType.getName(), FloodService.Type.CONSUMER_ET);
+        NakadiControllerAT.blockFlooder(flooder);
+
+        TestStreamingClient client = TestStreamingClient
+                .create(URL, subscription.getId(), "")
+                .start();
+        Thread.sleep(2000);
```

Comment:
We decided not to use these plain sleeps but rather wait to specific thing. 
Please take a look at other tests in the class. You can do something like
`waitFor(() -> assertThat(MoreStatus.TOO_MANY_REQUESTS.getStatusCode(), client.getResponseCode()));`